scripts/s3manage.py

```diff
@@ -451,14 +451,21 @@ def parse_arguments(argv):
             git_branch = argv[5]
 
     if cmd_type in ('activate', 'upload', 'info', 'delete'):
-        bucket_name = argv[3]
+        target_or_bucket_name = argv[3]
     elif cmd_type in ('list'):
-        bucket_name = argv[2]
+        target_or_bucket_name = argv[2]
+
+    # Legacy stuff for jenkins
+    if target_or_bucket_name in ('infra', 'int', 'prod'):
+        deploy_target = target_or_bucket_name
+        bucket_name = 'mf-geoadmin3-{}-dublin'.format(deploy_target)
+    else:
+        pattern = r".*-(int|prod|infra|dev)-.*"
+        match = re.match(pattern, bucket_name)
```

Comment:
I think that these 2 lines are completely useless
``` python
pattern = r".*-(int|prod|infra|dev)-.*"
match = re.match(pattern, bucket_name)
```
moreover you set `bucket_name` only at next line, I don't think this would run

Reply:
As long as not all branches are using the bucket name, we have to support both approaches.

Reply:
Not guessing anything anymore. Using variables instead.